tests/phpunit/CRM/Custom/Import/Parser/ApiTest.php

```diff
@@ -30,10 +30,10 @@ public function testImport(): void {
     $customGroupID = $this->ids['CustomGroup']['level'];
     $this->createDateCustomField(['date_format' => 'yy', 'custom_group_id' => $customGroupID])['id'];
     $this->importCSV('custom_data_date_select.csv', [
-      ['name' => 'contact_id'],
-      ['name' => 'Pick_Color'],
+      ['name' => 'Contact.id'],
```

Comment:
So this field is really `entity_id`, right?

Reply:
@colemanw yeah - - it's also supporting Contact.external_identifier - & dedupe fields now too I think